src/support/small_vector.h

```diff
@@ -65,13 +65,26 @@ template<typename T, size_t N> class SmallVector {
   using value_type = T;
 
   SmallVector() {}
+  SmallVector(const SmallVector<T, N>& other)
+    : usedFixed(other.usedFixed), fixed(other.fixed), flexible(other.flexible) {
+  }
+  SmallVector(SmallVector<T, N>&& other)
+    : usedFixed(other.usedFixed), fixed(std::move(other.fixed)),
+      flexible(std::move(other.flexible)) {}
   SmallVector(std::initializer_list<T> init) {
     for (T item : init) {
       push_back(item);
     }
   }
   SmallVector(size_t initialSize) { resize(initialSize); }
 
+  SmallVector<T, N>& operator=(const SmallVector<T, N>& other) {
```

Comment:
Should we add a move assignment operator for completeness?

Reply:
Hmm, perhaps let's wait until there is a user? Otherwise adding it now would be dead/untested code.

Reply:
The problem is that if someone does try to use move assignment, the compiler will gracefully fall back to using copy assignment and the user will have no idea that they didn't get the move they wanted. We can prevent that by providing a move assignment operator eagerly, and the code should be simple enough that there shouldn't be any problems. (Alternatively we could provide a move assignment whose body is a WASM_UNREACHABLE.)

Reply:
Good point, that does sound risky. Added.